src/hound/vcs/svn.go

```diff
@@ -0,0 +1,79 @@
+package vcs
+
+import (
+	"bytes"
+	"io"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+func init() {
+	RegisterVCS("svn", &SVNDriver{})
+}
+
+type SVNDriver struct{}
+
+func (g *SVNDriver) HeadHash(dir string) (string, error) {
+	cmd := exec.Command(
```

Comment:
Ah yes I just found the regex that was checking for a 40 character hash. The output of svnversion will give you the revision number and possible trailing characters if the [working copy is modified](http://svnbook.red-bean.com/en/1.7/svn.ref.svnversion.re.html). 

I didn't think using it would be a problem if the WC was never going to be modified.